addons/website_livechat/models/discuss_channel.py

```diff
@@ -58,7 +59,10 @@ def _get_visitor_history(self, visitor):
         :return: arrow separated string containing navigation history information
         """
         recent_history = self.env['website.track'].search([('page_id', '!=', False), ('visitor_id', '=', visitor.id)], limit=3)
-        return ' → '.join(visit.page_id.name + ' (' + visit.visit_datetime.strftime('%H:%M') + ')' for visit in reversed(recent_history))
+        return [
+            (visit.page_id.name, fields.Datetime.to_string(visit.visit_datetime))
+            for visit in reversed(recent_history)
```

Comment:
As this is stable, we should ideally keep the API intact, so keep sending a string in the same format as before.